alien4cloud/topology.go

```diff
@@ -517,3 +518,55 @@ func (t *topologyService) SaveA4CTopology(ctx context.Context, a4cCtx *TopologyE
 
 	return processA4CResponse(response, nil, http.StatusOK)
 }
+
+func (t *topologyService) GetAllTopologies(ctx context.Context) ([]BasicTopologyInfo, error) {
+
+	getTopoJSON, err := json.Marshal(
+		ListTopologiesCreateRequest{
+			from: 0,
+			query: "",
+			size: 0,
+		},
+	)
+
+	if err != nil {
+		return nil, errors.Wrap(err, "Cannot marshal an a4cgetTopologiesCreateRequest structure")
+	}
+
+	response, err := t.client.doWithContext(ctx,
+		"POST",
+		fmt.Sprintf("%s/catalog/topologies/search", a4CRestAPIPrefix),
+		[]byte(string(getTopoJSON)),
+		[]Header{
+			{
+				"Content-Type",
```

Comment:
Loïc introduced recently constants for headers, you could use:
```
[]Header{contentTypeAppJSONHeader, acceptAppJSONHeader},
```

Reply:
Done.